conjur.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
 
   gem.add_dependency 'activesupport'
-  gem.add_dependency 'conjur-api', '~> 4.20', '>= 4.20.1'
+  gem.add_dependency 'conjur-api', '~> 4.20'
```

Comment:
That was Alan's suggestion. The version it was checking for got reverted or something. It didn't exist when I was testing this.